### DISCUSSION TITLE
Module visible to all logged in user

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#7050] [Help]
I've created a module in common folder and I want this is visible to all logged-in user; 

![Schermata 2023-12-29 alle 18 32 35](https://github.com/serenity-is/Serenity/assets/14870650/b9924fa2-7540-4c44-a0db-9f097f8485ed)

I've set this permission on my xyzRow.cs:

```
[ConnectionKey("Default"), Module("Common"), TableName("[kc].[VDocumenti]")]
[DisplayName("Documenti"), InstanceName("Documents")]

[ReadPermission("")]
[ModifyPermission("")]
[NavigationPermission("")]
```

but when I click on link, an "Access Denied" error appear.

this is the link settings:
`[assembly: NavigationLink(1300, "Common/Documents", url: "~/Common/Documents", permission: "", icon: null)]`

What am I missing?
Thank you,
Elena.

**Comment by reach2rv** [comment]
I think you are missing "*" in navigation link permission.

**Comment by elenaaralla** [comment]
Hi @reach2rv,

yes; I've added "?" to permission on Row and link; 
"?"  is for "Logged user".
Thank you.
